src/main/java/net/earthcomputer/clientcommands/mixin/MixinGameRenderer.java

```diff
@@ -0,0 +1,23 @@
+package net.earthcomputer.clientcommands.mixin;
+
+import net.earthcomputer.clientcommands.render.RenderQueue;
+import net.minecraft.client.render.GameRenderer;
+import net.minecraft.client.util.math.MatrixStack;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(GameRenderer.class)
+public abstract class MixinGameRenderer {
+
+    @Inject(method = "renderWorld", at = @At(value = "INVOKE", target = "Lnet/minecraft/util/profiler/Profiler;swap(Ljava/lang/String;)V"))
+    private void renderWorldHand(float delta, long time, MatrixStack matrixStack, CallbackInfo ci) {
+        RenderQueue.InjectLoc.HAND.onRender(delta, time, matrixStack);
+    }
+
+    @Inject(method = "renderWorld", at = @At(value = "INVOKE", target = "Lnet/minecraft/util/profiler/Profiler;swap(Ljava/lang/String;)V"))
+    private void renderWorldCamera(float delta, long time, MatrixStack matrixStack, CallbackInfo ci) {
+        RenderQueue.InjectLoc.CAMERA.onRender(delta, time, matrixStack);
+    }
```

Comment:
Don't these both inject into the same location? I think INVOKE_STRING might be what you're looking for.

Reply:
Oh yea, you're right... missed that